.github/renovate.json5

```diff
@@ -6,6 +6,26 @@
     fileMatch: ["^tox.ini$", "(^|/)requirements([\\w-]*)\\.txt$"]
   },
    packageRules: [
+    {
+      // Internal package minor patch updates get top priority, with auto-merging
+      groupName: "internal package patch releases",
+      matchPackagePatterns: ["^craft-.*"],
+      matchUpdateTypes: ["minor", "patch", "pin", "digest"],
+      prPriority: 10,
+      automerge: true,
+      minimumReleaseAge: "0 seconds",
+      schedule: ["at any time"],
+    },
+    {
+      // Same as above, but for hotfix branches, only for patch, and without auto-merging.
+      groupName: "internal package patch releases (hotfix)",
+      matchPackagePatterns: ["^craft-.*"],
+      matchUpdateTypes: ["patch", "pin", "digest"],
+      prPriority: 10,
+      minimumReleaseAge: "0 seconds",
+      schedule: ["at any time"],
+      matchBaseBranches: ["/^hotfix/.*/"],  // All hotfix branches
```

Comment:
This is probably going to be noisy in repos where the old hotfix branches haven't been deleted (snapcraft).

I think this will remind us to remove them!